JS/Lesson31/onur/my-app/src/components/List/index.jsx

```diff
@@ -1,31 +1,49 @@
 import './styles.css';
-import {ListItem} from '../ListItem';
-import {useState} from 'react';
+import { ListItem } from '../ListItem';
+import { useState } from 'react';
+import { NewToDoItem } from '../NewToDoItem';
 
 export const List = () => {
   const data = [
-    {id: 1, title: "Do dishes", completed: false},
-    {id: 2, title: "Do homework", completed: false},
-    {id: 3, title: "Go running", completed: false},
-    {id: 4, title: "Do dishes 2", completed: false},
-    {id: 5, title: "Do homework 3", completed: false},
-    {id: 6, title: "Go running 4", completed: false},
+    { id: 1, title: "Do dishes", completed: false },
+    { id: 2, title: "Do homework", completed: false },
+    { id: 3, title: "Go running", completed: false },
+    { id: 4, title: "Do dishes 2", completed: false },
+    { id: 5, title: "Do homework 3", completed: false },
+    { id: 6, title: "Go running 4", completed: false },
   ];
 
   const [todoItems, setTodoItems] = useState(data);
 
+  const handleCreateTodo = (newItem) => {
+    const newTodoId = (todoItems.length) + 1;
```

Comment:
There's a problem here. Let's say there're 5 items in the list, ids are 1,2,3,4,5.
Then, I remove item 4 (item with id 4 is gone) and add a new item. The id of the new item will be 5, so the list would look like this: 1,2,3,5,5. 
We want id to be a unique value. Normally, you'd use a library to generate a uuid (Universal Unique Identifier) but you can use a hack here:
 ```const newTodoId = Math.max(...todoItems.map((item) => item.id)) + 1;```